test/functional/readconcern_tests.js

```diff
@@ -510,6 +559,73 @@ describe('ReadConcern', function() {
     }
   });
 
+  it('Should set majority readConcern option for aggregate command but ignore due to out', {
```

Comment:
This and the previous test are confusingly worded so as to seem like they are testing the same thing. It looks like this one checks read concern at the operation level, while the previous tests it set at the client level